modules/Makefile.circleci

```diff
@@ -45,6 +45,8 @@ circle\:tag-latest:
 	@$(SELF) docker:login
 	@echo "INFO: Tagging latest"
 	@$(SELF) DOCKER_TAG=latest docker:push
+	@git tag --force "$(CIRCLE_BRANCH)-docker-latest"
+	@git push origin --force "tags/$(CIRCLE_BRANCH)-docker-latest"
```

Comment:
It looks like`agent-desktop`, `supernova`, `help`, and `edge-broker` don't always do a `circle:tag` before `deploy-kubernetes`. We could make both `circle:tag-latest` and `circle:deploy-kubernetes` target dependent on `tag`. It could be extra chatty, but the docker push should be quick since nothing to do right?